### PR TITLE
fix profiler table name generation

### PR DESCRIPTION
**Pull request by iporollo**
## Description

Fixing #21 to generate correct table name strings. Example monitor that is profiled: 

```
monosi:
  monitors:
  - table: snowflake_sample_data.tpcds_sf10tcl.call_center
    timestamp_field: cc_rec_end_date
```

## Notes

This does affect the file names as well. That said, I believe this is better because it profiles more accurately now (previously if table names were the same in different schema, they would be overwritten)

image

### CODE REVIEW

**Comment by unkrich** [comment]
This doesn't fix the core issue. The profiler is not the actual problem. The problem is in parsing the name and resolving it to a fully qualified table name

**Review by unkrich — CHANGES_REQUESTED** [review]
See previous comment

File: monosi/profiler.py [hunk 1]
```diff
@@ -58,7 +58,10 @@ def from_metadata(cls, metadata):
                 name=row['COL_NAME'], 
                 data_type=resolve_to_type_from_str(row['COL_TYPE']),
             )
-            tables = cls._insert_in_table(tables, row['NAME'], column)
+            table_name_elements = [row['DATABASE'], row['SCHEMA'], row['NAME']]
```
Comment (unkrich) [inline monosi/profiler.py:61]: Not sure we should assume database and schema are for sure returned here
```
table_name_elements = list(filter(lambda x: x is not None, [row['DATABASE'], row['SCHEMA'], row['NAME']]))
```

**Comment by iporollo** [comment]
Closing for now, will re-open with updated changes + base branch

**Pull request closed without merging**